docs/development/commits-branches-merging.md

```diff
@@ -37,13 +37,13 @@ base.
 
 In addition to `main`, the repository has three other long-lived branches:
 
-- `test` and `prod` correspond to the Test and Production [deploy environments](../deployment/README.md), respectively.
+- `test` and `prod` are read-only (no longer actively used) and correspond to the Test and Production [deploy environments](../deployment/README.md), respectively.
 - `gh-pages` hosts the compiled documentation, and is always forced-pushed by the
   [docs build process](../getting-started/documentation.md#deploying).
 
 ### Protection rules
 
-[Branch protection rules][gh-branch-protection] are in place on three environment branches (`main`, `test`, `prod`) to:
+[Branch protection rules][gh-branch-protection] are in place on `main` to:
```

Comment:
Could you please add a small note about `test` and `prod` also being protected from deletion and marked as read-only as they are no longer used, somewhere in this section.